src/components/ItaliaTheme/Icons/Icon.jsx

```diff
@@ -11,8 +11,7 @@ import XTwitterSVG from './svg/XTwitterSVG';
 import { FontAwesomeIcon } from 'design-comuni-plone-theme/components/ItaliaTheme';
 
 const Icon = (props) => {
-  const { icon, className, color, size, padding, ...rest } = props;
-
+  const { icon, className, color, size, padding, title, ...rest } = props;
```

Comment:
Come visto per l'altra PR su aria-hidden, questa modifica è ridondante, perché se nelle props hai `title` e non lo estrapoli, quello rimane dentro `rest` e poi `rest` viene spreaddato in `FontAwesomeIcon` quindi è esattamente identico a passargli `title={title}`, quindi puoi ometterlo, non serve scriverlo.